### DISCUSSION TITLE
日志在循环报错（但是还能用）

### DISCUSSION THREAD

**Discussion by HYhuangyun** [xfgryujk/blivechat#209] [提问]
[blivechat.log](https://github.com/user-attachments/files/15784418/blivechat.log)
使用docker部署的
![QQ截图20240611161157](https://github.com/xfgryujk/blivechat/assets/106595395/3104b725-e8cb-4ed4-9ac6-4a8dfaf62a3b)
也没有开翻译啥的，求解

**Comment by HYhuangyun** [comment]
好吧，似乎并不能用
![QQ截图20240611171742](https://github.com/xfgryujk/blivechat/assets/106595395/95fa732f-fca7-4b04-adf9-5fa751765962)

**Comment by xfgryujk** [comment]
你Docker网络就没配好

**Reply by HYhuangyun** [reply]
> 你Docker网络就没配好

[https://github.com/xfgryujk/blivechat/discussions/194](url)请问在这个里面提到的关闭翻译要如何关闭，网络问题我解决了，谢谢

**Reply by xfgryujk** [reply]
> > 你Docker网络就没配好
> 
> [https://github.com/xfgryujk/blivechat/discussions/194](url)请问在这个里面提到的关闭翻译要如何关闭，网络问题我解决了，谢谢

配置文件里面`translator_configs`填空
